protected/humhub/modules/user/services/AuthClientService.php

```diff
@@ -64,10 +64,11 @@ public function getUser(): ?User
      * Updates (or creates) a user in HumHub using AuthClients Attributes
      * This method will be called after login or by cron sync.
      *
-     * @param User $user
-     * @return boolean succeed
+     * @param User|null $user
+     * @param bool $isOAuth
+     * @return bool succeed
      */
-    public function updateUser(User $user = null): bool
+    public function updateUser(User $user = null, bool $isOAuth = false): bool
```

Comment:
Reading the `isOAuth` param, please see the comment above.

Reply:
Ok